tutorials/networking/high_level_multiplayer.rst

```diff
@@ -388,9 +465,159 @@ have loaded the game scene.
         players.clear()
         server_disconnected.emit()
 
+ .. code-tab:: csharp
+
+    using Godot;
+
+    public partial class Lobby : Node
+    {
+        public static Lobby Instance { get; private set; }
+
+        // These signals can be connected to by a UI lobby scene or the game scene.
+        [Signal]
+        public delegate void PlayerConnectedEventHandler(int peerId, Godot.Collections.Dictionary<string, string> playerInfo);
+        [Signal]
+        public delegate void PlayerDisconnectedEventHandler(int peerId);
+        [Signal]
+        public delegate void ServerDisconnectedEventHandler();
+
+        private const int Port = 7000;
+        private const string DefaultServerIP = "127.0.0.1"; // IPv4 localhost
+        private const int MaxConnections = 20;
+
+        // This will contain player info for every player,
+        // with the keys being each player's unique IDs.
+        public Godot.Collections.Dictionary<long,Godot.Collections.Dictionary<string,string>> Players = new Godot.Collections.Dictionary<long,Godot.Collections.Dictionary<string,string>>();
```

Comment:
Since it's public, it should be a property. Although it doesn't seem to be accessed from the outside, so it could be private.

```suggestion
        public Godot.Collections.Dictionary<long, Godot.Collections.Dictionary<string, string>> Players { get; set; } = new Godot.Collections.Dictionary<long, Godot.Collections.Dictionary<string, string>>();
```